docs/core/entity/update.md

```diff
@@ -47,6 +47,26 @@ Supported features are defined by using values in the `UpdateEntityFeature` enum
 
 ## Methods
 
+### Compare versions
+
+This method should be implemented when needed to override the default version comparison logic:
+
+```python
+def version_is_newer(self, latest_version: str, installed_version: str) -> bool:
+        """Return True if available version is newer then installed version."""
+        return AwesomeVersion(
```

Comment:
Is this the default logic?

Reply:
Nope, is an example of how it can be overwritten. I'll make it more clear

Reply:
Maybe it would be good to show the default logic, so you know when you need something else?